File: src/java/ards-adif-api/src/main/java/com/ards/adif/api/model/RegionApplicability.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the region_applicability database table.
 * 
 */
@Entity
@Table(name="region_applicability")
@NamedQuery(name="RegionApplicability.findAll", query="SELECT r FROM RegionApplicability r")
public class RegionApplicability implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	//bi-directional many-to-one association to Region
	@ManyToOne
	@JoinColumn(name="region_id", nullable=false)
	private Region region;

	//bi-directional many-to-one association to Weblink
	@ManyToOne
	@JoinColumn(name="weblink_id", nullable=false)
	private Weblink weblink;

	public RegionApplicability() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Region getRegion() {
		return this.region;
	}

	public void setRegion(Region region) {
		this.region = region;
	}

	public Weblink getWeblink() {
		return this.weblink;
	}

	public void setWeblink(Weblink weblink) {
		this.weblink = weblink;
	}

}

File: src/dotnet/ArdsTools/Ards.Tools.Domain/Models/RegionApplicability.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.Tools.Domain.Adif.Adif.Models
{
    [Table("region_applicability", Schema = "adif")]
    public partial class RegionApplicability
    {
        [Column("id")]
        public int Id { get; set; }
        [Column("region_id")]
        public int RegionId { get; set; }
        [Column("weblink_id")]
        public int WeblinkId { get; set; }

        [ForeignKey("RegionId")]
        [InverseProperty("RegionApplicability")]
        public virtual Region Region { get; set; }
        [ForeignKey("WeblinkId")]
        [InverseProperty("RegionApplicability")]
        public virtual Weblink Weblink { get; set; }
    }
}
